client/src/components/pages/NewTourneyHome.js

```diff
@@ -63,6 +63,7 @@ function NewTourneyHome({ tourney, user, setUser, setLoginAttention }) {
         lobbyMaxSignups: data.lobbyMaxSignups || 8,
         blacklist: (data.blacklist || []).toString(),
         requiredCountries: data.requiredCountries || [],
+        discordServerId: data.discordServerId || "",
```

Comment:
how do you find the discord server id/can this default to GTS server?

Reply:
server id you can get by just right clicking a server and clicking "copy server id". I didn't really want to use a default ID cause I usually like to make things optional... but idk, i guess i could hard code in a default id, or maybe make it an input placeholder or display a note. up to you guys

Reply:
alright if it's that easy to get the id, then i guess we can leave it blank